GatherPress/gatherpress
# Issue 571: Added Maximum Attending Limit at the event level.

### Description of the Change

Benefits realized: Maximum Attending Limit is now on the event level

![Screen Shot 2024-03-01 at 2 45 03 AM (2)](https://github.com/GatherPress/gatherpress/assets/67053237/9eaba4cd-40d6-4109-b096-cf639bb163b2)



Closes #571 

### How to test the Change

In the event setting you will see a panel body labeled Maximum Attending Limit. Set the limit with the number control.

### Changelog Entry

> Added - New feature


### Credits

Props 
@bluecollarcoders 

### Checklist:
